modules/understanding-upgrade-channels.adoc

```diff
@@ -33,6 +33,11 @@ In addition to the stable channel, all even-numbered minor versions of {product-
 Both standard and non-EUS subscribers can access all EUS repositories and necessary RPMs (`rhel-*-eus-rpms`) to be able to support critical purposes such as debugging and building drivers.
 ====
 
+[IMPORTANT]
+====
+EUS channels are the only channels which receive additional z-streams while a release is in the EUS phase.
```

Comment:
```suggestion
EUS channels are the only channels that receive additional z-streams while a release is in the EUS phase.
```
- Use "that" for restrictive clauses and "which" for non-restrictive clauses. See https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases.
- Avoid placing two admonitions too closely together. Consider moving the less important admonition into the flow of the body text. See https://redhat-documentation.github.io/supplementary-style-guide/#admonitions.